From f3c68649807f90c961907675e8fca901ec0ea532 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 03 Sep 2021 18:41:38 +0300
Subject: [PATCH] [SW-0] gaudi2: pad boot straps regs values

When printing registers content it is more readable when the printed
value is of the size of a register.
Pad the boot strap printed values.

Change-Id: I2f87d52746561496e9c90df29d10e2eca490b6cc
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/common.c b/arch/gaudi2/common/common.c
index 86298c7..68bffb6 100644
--- a/arch/gaudi2/common/common.c
+++ b/arch/gaudi2/common/common.c
@@ -531,13 +531,13 @@
 {
 	u32 boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS_L);
 
-	preboot_puts_hex_raw("Boot strap pins low ", boot_strap, false);
+	preboot_puts_hex_raw("Boot strap pins low ", boot_strap, true);
 	preboot_puts("\n");
 	boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_BOOT_STRAP_PINS_H);
-	preboot_puts_hex_raw("Boot strap pins high ", boot_strap, false);
+	preboot_puts_hex_raw("Boot strap pins high ", boot_strap, true);
 	preboot_puts("\n");
 	boot_strap = regmap_readl(mmPSOC_GLOBAL_CONF_LEGACY_BOOT_STRAPS);
-	preboot_puts_hex_raw("Boot strap pins legacy ", boot_strap, false);
+	preboot_puts_hex_raw("Boot strap pins legacy ", boot_strap, true);
 	preboot_puts("\n");
 }
 
